discoveryv2/discovery_v2_integration_test.go

```diff
@@ -191,10 +191,10 @@ func TestDocument(t *testing.T) {
 
 	updateDocument, _, responseErr := service.UpdateDocument(
 		&discoveryv2.UpdateDocumentOptions{
-			ProjectID:    projectID,
-			CollectionID: collectionID,
-			DocumentID:   addDocument.DocumentID,
-			File:         fileInfo,
+			ProjectID:       projectID,
+			CollectionID:    collectionID,
+			DocumentID:      addDocument.DocumentID,
+			File:            fileInfo,
```

Comment:
did you confirm analyzedocument is working? some of the sdks have been having issues.